edgelet/iotedge/src/main.rs

```diff
@@ -270,6 +270,12 @@ fn run() -> Result<(), Error> {
                         .value_name("IOTHUB_HOSTNAME")
```

Comment:
At line 177, change the helptext to `"Output format. Note that JSON output contains some additional information like OS name, OS version, disk space, etc."`